sources/web/datalab/polymer/components/datalab-sidebar/datalab-sidebar.html

```diff
@@ -0,0 +1,72 @@
+<link rel="import" href="../../components/shared-styles/shared-styles.html">
+
+<link rel="import" href="../../bower_components/iron-selector/iron-selector.html">
+<link rel="import" href="../../bower_components/iron-icons/iron-icons.html">
+<link rel="import" href="../../bower_components/iron-icons/editor-icons.html">
+<link rel="import" href="../../bower_components/iron-icons/hardware-icons.html">
+<link rel="import" href="../../bower_components/iron-icons/av-icons.html">
+<link rel="import" href="../../bower_components/paper-button/paper-button.html">
+
+<dom-module id="datalab-sidebar">
+  <template>
+    <style include="datalab-shared-styles">
+      a, u {
+        text-decoration: none;
```

Comment:
Weren't you just complaining about how annoying it is to have invisible anchors? What's the intent here?